src/components/v1/index.ts

```diff
@@ -309,7 +309,7 @@ export default class ComponentV1 extends Component {
         (deployment_node as CloudNode<'deployment'>).inputs.services?.push({
           id: `\${{ ${service_node.id}.id }}`,
           account: `\${{ ${service_node.id}.account }}`,
-          port: `\${{ ${service_node.id}.target_port }}`,
+          port: `\${{ ${service_node.id}.port }}`,
```

Comment:
I don't think this is right. The target port is the port the application listens on. This is important for it to register itself with load balancers. The `port` value is supposed to be the value the `service` listens on before potentially re-writing when it passes the request to the downstream deployment/pods.

Reply:
Oh good catch this may have been a mistake when merging in changes - I at least don't remember a reason I updated this part so I'll verify that `\${{ ${service_node.id}.target_port }}` is doing the right thing